src/lib.rs

```diff
@@ -178,11 +187,22 @@ impl<T> ScopedKey<T> {
     }
 }
 
+/// test example from issue #23
+/// ```compile_fail
+/// use scoped_tls::scoped_thread_local;
+/// scoped_thread_local!(static VAR: u8);
+/// VAR.inner.with(|v| v.set(1 as *const ())); // completely safe
+/// VAR.with(|v| *v); // tries to read from address 1
+/// ```
+const _: () = ();
```

Comment:
It's ok to omit this since I don't think there's any risk of this "regressing" or coming back. Thanks though!

Reply:
so, should I remove that or is it fine to leave it in?

Reply:
This should be removed

Reply:
done